modules/siddhi-core/src/main/java/org/wso2/siddhi/core/query/processor/stream/function/StreamFunctionProcessor.java

```diff
@@ -42,6 +44,7 @@ protected void processEventChunk(ComplexEventChunk complexEventChunk, Processor
                     }
                     outputData = process(inputData);
                     complexEventPopulater.populateComplexEvent(complexEvent, outputData);
+                    break;
```

Comment:
Hi Damith,

Usually we don't need to add a break for the default case that comes at bottom. Was there any specific reason for you to add a break for the default?